.github/workflows/doc-tests.yaml

```diff
@@ -53,6 +53,27 @@ jobs:
           repository: 'gravitational/docs-website'
           path: 'docs'
 
+      - name: Generate GitHub Token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.REVIEWERS_APP_ID }}
+          private-key: ${{ secrets.REVIEWERS_PRIVATE_KEY }}
+
+      - name: Check out shared-workflows
+        uses: actions/checkout@v4
+        with:
+          repository: gravitational/shared-workflows
+          path: .github/shared-workflows
```

Comment:
I would recommend not putting this under `.github`. Github likes to introduce breaking changes without notification, and I wouldn't be surprised if they start treating this as a "special" directory at some point. Maybe `${{ runner.temp }}/shared-workflows`, or just `shared-workflows`?